src/FffPolygonGenerator.cpp

```diff
@@ -795,11 +795,12 @@ void FffPolygonGenerator::removeEmptyFirstLayers(SliceDataStorage& storage, cons
         log("Removing %d layers because they are empty\n", n_empty_first_layers);
         for (SliceMeshStorage& mesh : storage.meshes)
         {
+            const int deltaZ = layer_height_0 + layer_height * (n_empty_first_layers - 1);
```

Comment:
The difference in height between two layers never includes the height of the first layer.

If the first layer is printed on 1.0 and the second on 1.1 and the third on 1.2, but the first two layers are empty then the third layer should be printed on 1.0, so the difference is `n_empty_first_layers * layer_height` not `layer_height_0 + layer_height * (n_empty_first_layers - 1)`

Reply:
Understood.

Now, why do you say that the problem this is addressing is solved in another way?